react/LocaleSwitcher.tsx

```diff
@@ -66,7 +66,7 @@ const LocaleSwitcher = () => {
   const [openLocaleSelector, setOpenLocaleSelector] = useState(false)
 
   const supportedLanguages =
-    data?.currentBinding.supportedLocales ?? data?.languages.supported ?? []
+    data?.currentBinding?.supportedLocales ?? data?.languages.supported ?? []
```

Comment:
Add to languages too maybe?

```suggestion
    data?.currentBinding?.supportedLocales ?? data?.languages?.supported ?? []
```

Reply:
I don't think that would happen, but it doesn't hurt hahahah